lib/api/src/lib/stories.ts

```diff
@@ -183,16 +186,19 @@ export const transformStoryIndexToStoriesHash = (
   { provider }: { provider: Provider }
 ): StoriesHash => {
   const countByTitle = countBy(Object.values(index.stories), 'title');
-  const input = Object.entries(index.stories).reduce((acc, [id, { title, name, importPath }]) => {
-    const docsOnly = name === 'Page' && countByTitle[title] === 1;
-    acc[id] = {
-      id,
-      kind: title,
-      name,
-      parameters: { fileName: importPath, options: {}, docsOnly },
-    };
-    return acc;
-  }, {} as StoriesRaw);
+  const input = Object.entries(index.stories).reduce(
+    (acc, [id, { title, name, importPath, parameters }]) => {
+      const docsOnly = name === 'Page' && countByTitle[title] === 1;
+      acc[id] = {
+        id,
+        kind: title,
+        name,
+        parameters: { fileName: importPath, options: {}, docsOnly, ...parameters },
```

Comment:
@tmeasday could `...parameters` ever override `options` or `fileName`? if so, would that be an issue?

Reply:
I think `fileName` will always be the same. I think we just define options here because other code assumes it exists, I don't really remember why.